core-api-library/library/src/main/java/net/gini/android/core/api/MediaTypes.java

```diff
@@ -1,9 +1,12 @@
 package net.gini.android.core.api;
 
-
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+/**
+ *  Media Types used across the application (e.g document types, http headers)
+ */
+
```

Comment:
[Request] Please remove the empty line here. From what I've seen the convention is to have the comment block right above class/method/property.